### PR TITLE
#8915 - Hide delete button

### LINKED ISSUE #8915: Temporarily disable the deletion of event groups
### Problem Description

Deleting event groups currently leads to the user session idling and the database transaction not executing (#8851). Unfortunately, this is not easily fixable. In order to not let users run into this problem, we need to temporarily disable the deletion of event groups until we have found a proper fix for the issue.

### Proposed Change

Hide the "Delete" button from the event group form.

### PR DESCRIPTION
**Pull request by MateStrysewske**
Fixes #8915

### CODE REVIEW

File: sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventGroupController.java [hunk 1]
```diff
@@ -203,19 +202,21 @@ public CommitDiscardWrapperComponent<?> getEventGroupEditComponent(String uuid)
 			});
 		}
 
-		if (user.hasUserRight(UserRight.EVENTGROUP_DELETE) && hasRegion) {
-			editView.addDeleteListener(() -> {
-				deleteEventGroup(eventGroup);
-				UI.getCurrent().getNavigator().navigateTo(EventsView.VIEW_NAME);
-			}, I18nProperties.getString(Strings.entityEventGroup));
-		}
+		// TODO Temporarily deleted in #8915, enable again in #8851
```
Comment (StefanKock) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventGroupController.java:205]: Write this marker like this:
"TODO #8851: Enable temporarily deleted button when EventGroup deletion works"

**Review by StefanKock — CHANGES_REQUESTED** [review]

File: sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventGroupController.java [hunk 2]
```diff
@@ -203,19 +202,21 @@ public CommitDiscardWrapperComponent<?> getEventGroupEditComponent(String uuid)
...
+//			}, I18nProperties.getString(Strings.entityEventGroup));
+//		}
 
 		// Initialize 'Archive' button
 		if (user.hasUserRight(UserRight.EVENTGROUP_ARCHIVE) && hasRegion) {
 			boolean archived = FacadeProvider.getEventGroupFacade().isArchived(uuid);
-			Button archiveEventButton = ButtonHelper.createButton(archived ? Captions.actionDearchiveInfrastructure : Captions.actionArchiveInfrastructure, e -> {
-				archiveOrDearchiveEventGroup(uuid, !archived);
-			}, ValoTheme.BUTTON_LINK);
+			Button archiveEventButton =
```
Comment (StefanKock) [inline sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventGroupController.java:216]: Better not format something else: When you keep it clean, a revert action can work.

Comment (MateStrysewske) [reply sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventGroupController.java:216]: This has been done by automatic formatting; manually re-formatting it in the wrong way doesn't seem like a good idea to me.

Comment (StefanKock) [reply sormas-ui/src/main/java/de/symeda/sormas/ui/events/EventGroupController.java:216]: Just disable the automatic formatting for this change to not change something unintended. That's the problem when not everyone formats the code the same way :-(

**Review by StefanKock — APPROVED** [review]
Please only one change commit when merging this: amend, fixup or squash

**Pull request merged**